### ISSUE TITLE
LP File Finder

### ISSUE THREAD

**Issue by thanneken** [CenterForDigitalHumanities/SpectralRTI_Toolkit#74]
Just ran through everything and it mostly worked great. One small bug that I suspect is rather small...

I followed the procedure to generate a new LP file, rather than using one of the recycled ones. The first time I did so it threw an error that it couldn't find an LP file. Then I thought maybe it wanted the lp file in LightPositionData rather than a folder down in assembly-files (which itself would have been a small bug). 

When I ran it again it asked me to choose between two, but the way it was formatted (I think) gives a clue to the original bug. The prompt was:
```
We found light position files in your project directory. Please choose the source file to use from below.
File:
o /home/thanneken/Downloads/Ambrosiana_C73inf_115/LightPositionData/Sphere.lp
o /home/thanneken/Downloads/Ambrosiana_C73inf_115/LightPositionData/assembly-files//home/thanneken/Downloads/Ambrosiana_C73inf_115/LightPositionData/assembly-files/Sphere.lp
```
Note the `//` and that the path basically repeats for the second option. 

The bottom line is that it should have found the lp file in the first place at  `/home/thanneken/Downloads/Ambrosiana_C73inf_115/LightPositionData/assembly-files/Sphere.lp`

**Comment by thehabes** [comment]
tested with jar in repo, should be fixed now.

**Comment by thanneken** [comment]
Confirmed on Linux
